content/intro-to-storybook/index.md

```diff
@@ -27,20 +27,20 @@ authors:
     {
       src: "https://avatars2.githubusercontent.com/u/132554",
       name: "Tom Coleman",
-      detail: "Storybook core maintainer",
+      detail: "Storybook core",
     },
     {
       src: "https://avatars2.githubusercontent.com/u/263385",
       name: "Dominic Nguyen",
-      detail: "Storybook designer",
+      detail: "Storybook design",
     },
   ]
 contributors:
   [
     {
       src: "https://avatars2.githubusercontent.com/u/22988955",
       name: "Jonnie Bigodes",
```

Comment:
@domyen i was going to modify this, but it slipped my mind. Can you please change `Jonnie Bigodes` to João Cardoso? That's my actual name.  The username "jonniebigodes  or "Jonnie bigodes" is a nickname that was given to me when i was a kid  and kinda stucked 😄